### PR TITLE
=act correct two comments in AbstractNodeQueue.java

### PR DESCRIPTION
**Pull request by rkuhn**
cc @nomagic @nitsanw

### CODE REVIEW

**Comment by rkuhn** [comment]
(see also https://github.com/akka/akka/pull/15596#issuecomment-50446085)

File: akka-actor/src/main/java/akka/dispatch/AbstractNodeQueue.java [hunk 1]
```diff
@@ -14,7 +14,8 @@
  */
 @SuppressWarnings("serial")
 public abstract class AbstractNodeQueue<T> extends AtomicReference<AbstractNodeQueue.Node<T>> {
-    // Extends AtomicReference for the "head" slot (which is the one that is appended to) since Unsafe does not expose XCHG operation intrinsically
+    // Extends AtomicReference for the "head" slot (which is the one that is appended to) since
+    // Unsafe does not expose XCHG operation intrinsically before JDK 8
```
Comment (viktorklang) [inline akka-actor/src/main/java/akka/dispatch/AbstractNodeQueue.java]: Since 2.4 is JDK8+ this would no longer be an issue, tho?

Comment (rkuhn) [inline akka-actor/src/main/java/akka/dispatch/AbstractNodeQueue.java]: Well, the comment is still correct :-)

But what would we gain by changing?

Comment (viktorklang) [inline akka-actor/src/main/java/akka/dispatch/AbstractNodeQueue.java]: I guess what we gain is that extenders no longer can mess around with the full AR interface :)

Comment (rkuhn) [inline akka-actor/src/main/java/akka/dispatch/AbstractNodeQueue.java]: That is true. But I’m not yet convinced that this is a strong enough reason to change.

Comment (viktorklang) [inline akka-actor/src/main/java/akka/dispatch/AbstractNodeQueue.java]: That makes two of us :)

**Comment by rkuhn** [comment]
This can be merged by whoever sees the green button (as a token of approval).

**Pull request merged**